Gopkg.toml

```diff
@@ -80,3 +80,7 @@ required = [
 [[override]]
   revision = "411b2483e5034420675ebcdd4a55fc76fe5e55cf"
   name = "k8s.io/kube-openapi"
+
+[[constraint]]
```

Comment:
I'm not sure about adding this dep. Any way around it please?

Reply:
Fixed - I was just using the type it turns out which was pretty wasteful, especially since it was just a simple map.